core/src/main/java/nl/nn/adapterframework/parameters/ParameterValueList.java

```diff
@@ -95,22 +85,28 @@ public boolean contains(String name) {
 		return map.containsKey(name);
 	}
 
+	/** Performance intensive */
 	public ParameterValue remove(String name) {
```

Comment:
Je kan ook de hele method weghalen. Hij wordt nergens gebruikt

Reply:
Hij wordt in de SAP module gebruikt.

Reply:
hm, ja, in een mixed use van by name en by position.